index.js

```diff
@@ -15,11 +15,14 @@ function makeAMD(moduleContents, opts) {
       defines.push(define);
     }
   });
-  return 'define([' + includes.join(',') + '], ' +
+
+  var moduleName = opts.amdName ? '"' + opts.amdName(filePath) + '", ' : '';
```

Comment:
The option needs to be changed to `name` here & in the tests as well. Sorry for not mentioning that before.